pkg/virt-controller/watch/migration/migration_test.go

```diff
@@ -756,7 +767,7 @@ var _ = Describe("Migration watcher", func() {
 		})
 
 		It("should create another target pods if there is only one outbound migration on the node", func() {
-			// It should create a pod for this one
+			// It should not create a pod for this one
```

Comment:
? Look like we do expect the creation of a new pod, what am I missing?

Reply:
Might have accidentally changed the wrong comment here, let me restore it.